machine_learning/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(machine_learning)
+
+find_package(catkin REQUIRED)
+
+# catkin_python_setup()
+
+## Generate messages in the 'msg' folder
+# add_message_files(
+#   FILES
+# )
+
+## Generate services in the 'srv' folder
+# add_service_files(
+#   FILES
+# )
+
+## Generate added messages and services with any dependencies listed here
+# generate_messages(
+#   DEPENDENCIES
+#   std_msgs  # Or other packages containing msgs
+# )
+
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES machine_learning
+#  CATKIN_DEPENDS other_catkin_pkg
+#  DEPENDS system_lib
+)
```

Comment:
I like to leave in the lines that could possibly be used in the future, that way if someone adds a c++ file or wants messages, they can just uncomment the lines rather then going to some other cmake file and copying what they need